docs/node-operators/getting-started.mdx

```diff
@@ -25,54 +25,61 @@ This documentation is for the **Mainnet** release.
 
 ## Requirements
 
-**Software**: macOS, Linux (currently supports Debian 9 and Ubuntu 18.04 LTS), or any host with Docker
+**Software**: Supported environments include macOS, Linux (Debian 10, 11, 12 and Ubuntu 20.04 LTS), and any host machine with Docker. 
 
-:::note
-
-Windows is not officially supported at this time. However, it may be possible to use the [Docker instructions](./#docker). If you are interested in receiving a grant to develop Mina for Windows, please email grants@o1labs.org or reach out in the [Mina Protocol Discord server](https://discord.gg/minaprotocol).
+**Processor**: Only AMD64 CPU architecture is supported.
```

Comment:
@amc-ie  is processor the right term here? I didn't want to include AMD64 in the software section